collections/validator/check-if-a-object-is-empty.md

```diff
@@ -0,0 +1,8 @@
+~~~ javascript
+const isEmpty = object => Object.keys(object).length === 0
+
+// Examples
+isEmpty({a:1});      // false
+isEmpty({});         // true
+issEmpty({x: undefined})  // true
+~~~
```

Comment:
Can you remove this file as we already had it at https://1loc.dev/#check-if-an-object-is-empty?

Reply:
OK